bot/cogs/timeleft.py

```diff
@@ -0,0 +1,23 @@
+from discord.ext import commands

```

Comment:
Please format your imports according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports). Flake 8 should notify you of this, and most of the modern IDEs & code editors should have a command to do this for you.